continuous-integration/setup.sh

```diff
@@ -12,6 +12,7 @@ pacman -Syu --noconfirm --needed \
        file \
        flake8 \
        grep \
+       intel-compute-runtime \
```

Comment:
Did you intend to commit this in the "Bump Zivid SDK version to 1.4.0" commit?

Reply:
No it should be part of 9416b00cd0d035e59e89902f6148c0c0acb02035, will fix.

Reply:
Fixed